hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/routes/BaseRoute.java

```diff
@@ -16,8 +16,7 @@
 public abstract class BaseRoute extends RouteBuilder {
 	protected static final String JMS_DESTINATION_NAME_FORMAT = "queue:///%s?targetClient=1&&mdWriteEnabled=true";
 	
-	protected static final String MQ_URL_FORMAT = "jmsComponent:queue:%s?exchangePattern=InOut&replyTo=queue:///%s&replyToType=Shared&allowAdditionalHeaders=JMS_IBM_MQMD_MsgId";
-	
+	protected static final String MQ_URL_FORMAT = "jmsComponent:queue:%s?exchangePattern=InOut&replyTo=queue:///%s&replyToType=Exclusive&allowAdditionalHeaders=JMS_IBM_MQMD_MsgId&deliveryMode=1&requestTimeout=7000&explicitQosEnabled=true&timeToLive=7000";
```

Comment:
Both JMSDeliveryMode in the exchange header and deliveryMode in the url are set. Is this required? It mentions in the documentation that one of these takes priority based on the setting of the preserveMessageQos. I don't think the header will be considered as this property has a default of 'false'.